### ISSUE TITLE
Introduce sent indexing urls cache?

### ISSUE THREAD

**Issue by D39-Dev** [goenning/google-indexing-script#38]
Hi, thanks for your tool!
For the large websites, like 5k+ pages, is it possible to introduce the sent urls for indexing cache? 
For most of us is ok to run your tool on a daily basis in order to cover all the website pages with a daily limitation of 200 request.
Just needs to be sute that tool is not sending already sent pages.
Thanks!

**Comment by antoinekm** [comment]
Yes, I'd already thought of that, the problem is that there are going to be 2 schools:
- The first will want to double-check the urls whose indexing has already been requested, because they may have already been indexed.
- The second will want to save in the cache the urls whose indexing has been requested.

My solution would be not to recheck the urls until all of them have been requested to be indexed.

**Comment by goenning** [comment]
The usage of `getPublishMetadata` already prevents sending the same URL twice

**Comment by antoinekm** [comment]
> The usage of `getPublishMetadata` already prevents sending the same URL twice

Yes, but the problem is that there's a rate limit here too 🥲
